.github/deploy/review.hcl

```diff
@@ -25,6 +25,9 @@ job "blockbot-[[.environment_slug]]" {
         data        = <<EOF
 TOKEN={{ key "blockbot-dev/discord/token" }}
 DEBUG=false
+LDAP_USERNAME={{ key "blockbot-dev/ldap/username" }}
+LDAP_PASSWORD={{ key "blockbot-dev/ldap/password" }}
+UID_MAP={{ key "blockbot-dev/ldap/uid_map" }}
```

Comment:
same thing here, if the uid_map is discord uid's, I think then it should be under `blockblot/discord/uid_map` rather than under ldap cause that might cause confusion in the future with future admins/webgroup members